flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/ExecutionConfigOptions.java

```diff
@@ -19,12 +19,12 @@
 package org.apache.flink.table.api;
```

Comment:
Move this to `org.apache.flink.table.api.config`?

Reply:
+1